source/replication/replication-overview.rst

```diff
@@ -63,126 +63,51 @@ replication while adding the following MinIO-only features:
-       replication queue and adds them to the queue as space is available.
-
-       For multi-site replication, objects remain
-       in the ``PENDING`` state until replicated to *all* configured
-       remotes for that bucket or bucket prefix.
-
-   * - ``COMPLETED``
-     - The object has successfully replicated to the remote cluster.
-
-   * - ``FAILED``
-     - The object failed to replicate to the remote cluster. 
-
-       MinIO continuously scans for ``FAILED`` objects not yet in the
-       replication queue and adds them to the queue as space is available.
-
-   * - ``REPLICA``
-     - The object is itself a replica from a remote source.
-
-       MinIO ignores ``REPLICA`` objects, including any metadata-only changes
-       to that object. MinIO does not support 
-       AWS `replica modification sync 
-       <https://aws.amazon.com/about-aws/whats-new/2020/12/amazon-s3-replication-adds-support-two-way-replication/>`__
-       at this time.
-
-The replication process generally has one of the following flows:
-
-- ``PENDING -> COMPLETED``
-- ``PENDING -> FAILED -> COMPLETED``
-
-Synchronous vs Asynchronous Replication
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-MinIO supports specifying either asynchronous (default) or synchronous
-replication for a given remote target.
-
-With the default asynchronous replication, MinIO completes the originating
-``PUT`` operation *before* placing the object into a :ref:`replication queue
-<minio-replication-process>`. The originating client may therefore see a 
-successful ``PUT`` operation *before* the object is replicated. While
-this may result in stale or missing objects on the remote, it mitigates
-the risk of slow write operations due to replication load.
-  
-With synchronous replication, MinIO attempts to replicate the object *prior* to
-completing the originating ``PUT`` operation. MinIO returns a successful ``PUT``
-operation whether or not the replication attempts succeeds. While this may
-result in more reliable synchronization between the source and remote target,
-it may also increase the time of each write operation due to replication load.
-
-You must explicitly enable synchronous replication when configuring the remote
-target target using the :mc-cmd:`mc admin bucket remote add` command with the
-:mc-cmd-option:`~mc admin bucket remote add sync` flag.
-
 .. _minio-replication-behavior-resync:
 
-Resynchronization
-~~~~~~~~~~~~~~~~~
+Resynchronization (Disaster Recovery)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The :mc-cmd:`mc replicate resync` command can reset replication synchronization
-for a remote target. The resynchronization process attempts to queue all
-objects on the source bucket regardless of their 
-:ref:`replication status <minio-replication-process>`. Resynchronization
-primarily supports recovery after partial or total loss of a remote 
-replication target.
+The :mc-cmd:`mc replicate resync` command completely resynchronizes a specified
```

Comment:
@poornas did some minor clean up here, but not sure if the meaning is correct.

Reply:
this operation resyncs all objects in the bucket that match a replication rule. Also assuming here that existing object replication is enabled on the rule in question.

Reply:
Thanks, will add that info.